apps/aehttp/priv/oas3.yaml

```diff
@@ -1912,6 +1912,40 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /debug/check-tx/pool/{hash}:
+    get:
+      tags:
+        - internal
+        - debug
+      operationId: GetCheckTxInPool
+      description: "Check if a transaction in the pool can be merged or it is blocked by something: not enough tokens, missing nonce or something else"
+      parameters:
+        - $ref: '#/components/parameters/intAsString'
+        - $ref: '#/components/parameters/txHash'
+      responses:
+        "200":
+          description: The transaction can be included or is already included
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                required:
+                  - status
+        "400":
```

Comment:
Hmm, is `400` really the appropriate code for this?
If we're using the 400 range, these are [the standard interpretations](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml):

400 | Bad Request | [RFC7231, Section 6.5.1](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1)
401 | Unauthorized | [RFC7235, Section 3.1](https://datatracker.ietf.org/doc/html/rfc7235#section-3.1)
402 | Payment Required | [RFC7231, Section 6.5.2](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.2)
403 | Forbidden | [RFC7231, Section 6.5.3](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3)
404 | Not Found | [RFC7231, Section 6.5.4](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.4)
405 | Method Not Allowed | [RFC7231, Section 6.5.5](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.5)
406 | Not Acceptable | [RFC7231, Section 6.5.6](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.6)
407 | Proxy Authentication Required | [RFC7235, Section 3.2](https://datatracker.ietf.org/doc/html/rfc7235#section-3.2)
408 | Request Timeout | [RFC7231, Section 6.5.7](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.7)
409 | Conflict | [RFC7231, Section 6.5.8](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.8)
410 | Gone | [RFC7231, Section 6.5.9](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.9)
411 | Length Required | [RFC7231, Section 6.5.10](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.10)
412 | Precondition Failed | [RFC7232, Section 4.2](https://datatracker.ietf.org/doc/html/rfc7232#section-4.2) [RFC8144, Section 3.2](https://datatracker.ietf.org/doc/html/rfc8144#section-3.2)
413 | Payload Too Large | [RFC7231, Section 6.5.11](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.11)
414 | URI Too Long | [RFC7231, Section 6.5.12](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.12)
415 | Unsupported Media Type | [RFC7231, Section 6.5.13](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.13) [RFC7694, Section 3](https://datatracker.ietf.org/doc/html/rfc7694#section-3)
416 | Range Not Satisfiable | [RFC7233, Section 4.4](https://datatracker.ietf.org/doc/html/rfc7233#section-4.4)
417 | Expectation Failed | [RFC7231, Section 6.5.14](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.14)
418-420 | Unassigned |  
421 | Misdirected Request | [RFC7540, Section 9.1.2](https://datatracker.ietf.org/doc/html/rfc7540#section-9.1.2)
422 | Unprocessable Entity | [RFC4918](https://datatracker.ietf.org/doc/html/rfc4918#section-11.2)
423 | Locked | [RFC4918](https://datatracker.ietf.org/doc/html/rfc4918#section-11.3)
424 | Failed Dependency | [RFC4918](https://datatracker.ietf.org/doc/html/rfc4918#section-11.4)
425 | Too Early | [RFC8470](https://datatracker.ietf.org/doc/html/rfc8470#section-5.2)
426 | Upgrade Required | [RFC7231, Section 6.5.15](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.15)
427 | Unassigned |  
428 | Precondition Required | [RFC6585](https://datatracker.ietf.org/doc/html/rfc6585#section-3)
429 | Too Many Requests | [RFC6585](https://datatracker.ietf.org/doc/html/rfc6585#section-4)
430 | Unassigned |  
431 | Request Header Fields Too Large | [RFC6585](https://datatracker.ietf.org/doc/html/rfc6585#section-5)
432-450 | Unassigned |  
451 | Unavailable For Legal Reasons | [RFC7725](https://datatracker.ietf.org/doc/html/rfc7725#section-3)

It's hard to find _any_ 400-range code that feels appropriate, since the return code descriptions describe different ways in which the request couldn't be served. In this case, it could, arguably: we found the tx, but it is blocked. This should be a response in the 200 range.

Reply:
Perhaps [202 Accepted](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.3) would be a better return code?

> The 202 (Accepted) status code indicates that the request has been
   accepted for processing, but the processing has not been completed.
   The request might or might not eventually be acted upon, as it might
   be disallowed when processing actually takes place.  There is no
   facility in HTTP for re-sending a status code from an asynchronous
   operation.
>
> The 202 response is intentionally noncommittal. [...]